README.md

```diff
@@ -193,6 +192,7 @@ mvn clean deploy -P release
 
 ## History
 
+- Version 1.7.0: Replace the internally used HTTP Client from Apache httpclient5 to java.net.http (needs Java 11)

```

Comment:
For completeness of the commit/merge you could also set the actual `<version>` in `pom.xml` to 1.7.0 😅